examples/tls-authentication/nginx/README.md

```diff
@@ -0,0 +1,73 @@
+# TLS termination
```

Comment:
Hum, I think that inside the auth directory is better. Will change here as suggested.